docs/platform/longview/longview-app-for-apache-classic/index.md

```diff
@@ -0,0 +1,322 @@
+-   **\<Yes\>**: the Longview tool will attempt to enable mod\_status, set the status page location, and restart Apache. This option is easier, but has the potential to disrupt your current Apache configuration. If you choose yes, and the configuration is successful, you should see output like the following:
+
+    {{< output >}}
+        [ ok ] Stopping Longview Agent: longview.
+        Checking Apache configuration...
+        Enabling module status.
+        To activate the new configuration, you need to run:
+          service apache2 restart
+        [....] Reloading web server config: . ok
+        Apache mod_status enabled
+        [ ok ] Starting Longview Agent: longview.
+        update-rc.d: using dependency based boot sequencing
+    {{</ output >}}
+
+Refresh the Longview Apache tab in the Linode Manager to verify that it's working now.
+
+If instead you receive a failure message, such as:
+
+  {{< output >}}
+    [FAIL] Reloading web server config: apache2 failed!
+  {{</ output >}}
+
+You will need to double-check your Apache installation, and then do a [manual configuration](#manual-configuration-all-distributions).
+
+### Unable to Access Local Server Status for Apache
+
+More specifically, the error will state `Unable to access local server status for Apache at <http://example.com/example?auto>: <error>:`. This error occurs when either:
+
+1.  Apache's `mod_status` setting is disabled or has been changed from the default location.
+
+2.  An Apache virtual host configuration is interfering with web requests to the `mod_status` location.
+
+ {{< note >}}
+This error occurs when Longview attempts to check the status page `location` listed in `/etc/linode/longview.d/Apache.conf`, or the default page at `127.0.0.1/server-status?auto`, but receives a non-200 HTTP response code. Basically, it means that the status page Longview is expecting is not being returned by the server.
+{{< /note >}}
+
+To fix this, follow these steps:
+
+1.  Make sure Apache is running:
+
+    Debian and Ubuntu:
+
+        service apache2 restart
+
+    CentOS and Fedora:
+
+        apachectl restart
+
+2.  Make sure `mod_status` is enabled. See the [Apache website](http://httpd.apache.org/docs/2.2/mod/mod_status.html#enable) for details. You can also check the output of the following command:
+
+        apachectl -M | grep status
+
+3.  Check the location for `mod_status`. The default location on Debian and Ubuntu systems is `http://127.0.0.1/server-status?auto` on localhost. In the Apache configuration file (typically `httpd.conf` or `status.conf`), this is designated with the lines:
+
+    {{< file "httpd.conf" apache >}}
+<Location /server-status>
+    SetHandler server-status
+
+{{< /file >}}
+
+    The `SetHandler server-status` line indicates that this is the location block for mod\_status. The location line itself sets the location.
```

Comment:
mod_status in backticks

Reply:
this is classic manager guide...so disregarding